apps/users/models.py

```diff
@@ -93,7 +99,7 @@ def __repr__(self):
 user_list_view = Table('user_list_view', metadata_obj,
                        Column('user_id', INTEGER),
                        Column('login', VARCHAR(50)),
-                       Column('last_visit', TIMESTAMP),
+                       Column('last_visit_at', TIMESTAMP),
```

Comment:
why don't you re-define type as DateTime?

Reply:
Left it as it was before, and now it's working because column type is defined in main.sql. But of course the problem will raise after migrations appears. I will change it.